hack/init-environment.sh

```diff
@@ -287,21 +290,23 @@ function main () {
     apply_terraform ${env_dir}
   fi
 
-  # Set up GKE namespaces.
+  # Initialize GKE Configurations.
   local -r kubeconfig_dir=${env_dir}/${KUBECONFIG_DIR_NAME}
   local -r command_center_config=${kubeconfig_dir}/command-center
   local -r processing_configs_dir=${kubeconfig_dir}/processing
 
+  # Initialize command-center GKE and services.
   create_namespaces ${command_center_config} ${COMMAND_CENTER_NAMESPACES[@]}
-  for kubeconfig in ${processing_configs_dir}/*; do
-    create_namespaces ${kubeconfig} ${PROCESSING_NAMESPACES[@]}
-  done
-
-  # Install command-center services.
   install_flux ${command_center_config} ${env_dir}
   install_secrets_manager ${command_center_config} ${env_dir} ${env}
   install_cloudsql_proxy ${command_center_config} ${env_dir} ${env}
   install_charts
+
+  # Initialize processing GKEs and services.
+  for kubeconfig in ${processing_configs_dir}/*; do
+    create_namespaces ${kubeconfig} ${PROCESSING_NAMESPACES[@]}
+    install_secrets_manager ${kubeconfig} ${env_dir} ${env}
+  done
```

Comment:
This is me reorganizing the flow:
1) Configs for command-center + processing clusters
2) All of the set up for command-center
3) All of the set up for the processing clusters